sprotty-elk/src/elk-layout.ts

```diff
@@ -247,6 +247,18 @@ export class ElkLayoutEngine implements IModelLayoutEngine {
         }
         sedge.routingPoints = points;
 
+        if (elkEdge.junctionPoints) {
+            sedge.children = sedge.children ?? [];
+            elkEdge.junctionPoints.forEach((jp, i) => {
+                const sJunction = <SNodeSchema> {
+                    type: 'junction',
+                    id: elkEdge.id + "_j" + i,
+                    position: jp
+                };
+                sedge.children?.push(sJunstion)
```

Comment:
This may not always be the desired behavior, as not all diagrams support elements of type `junction`.  Please add a protected method `createJunction` that does nothing by default. Then you can implement this junction point creation in a custom subclass for your application.